assets/agenda/components/LocationFilter.tsx

```diff
@@ -315,9 +315,9 @@ export class LocationFilter extends React.Component<any, any> {
      */
     renderRegionSearchResult(item: any, index: any) {
         const {selectedIndex} = this.state;
-
-        if (item.type === LOCATION_TYPE.CITY) {
-            return (
+        const enabledOptions = get(this.props, 'locationEnabledOptions');
+        if (item.type === LOCATION_TYPE.CITY && enabledOptions.city) {
```

Comment:
would you still need to check for `enabledOptions` if the server doesn't return those items?